Makefile

```
@@ -48,6 +48,9 @@ docker:
 docker-clean:
 	docker rmi -f '$(DOCKER_IMAGE)'
 
+lint: node-deps
+	npx eslint . --max-warnings=0 || exit 0
```

Can we change this to use the npm runner? I think the following will work.

```suggestion
	npm run lint --max-warnings=0 || exit 0
```